step-templates/f5-ltm-rest-enable-disable-member.json

```diff
@@ -0,0 +1,114 @@
+{
+  "Id": "4703dd4c-9f2d-471d-b15a-c1e345378695",
+  "Name": "F5-LTM-REST - Enable, Disable, or Force Offline Member",
+  "Description": "F5-LTM-REST - Enable, Disable, or Force Offline Member with optional wait for connections to drop",
+  "ActionType": "Octopus.Script",
+  "Version": 17,
```

Comment:
Please set to 1

Reply:
set version to 1 in commit 6eb13933058e106ceb006647323423f139ed8f13